<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1058</width>
    <height>667</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="src_img">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>351</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="copy_img">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>100</y>
      <width>351</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MS Shell Dlg 2</family>
     </font>
    </property>
    <property name="text">
     <string>Original</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>CÃ³pia</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_img_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>70</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Abrir imagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="copy_img_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>110</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Copiar imagem</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_x_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>170</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotacionar X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_y_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>200</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rotacionar Y</string>
    </property>
   </widget>
   <widget class="QPushButton" name="luminance_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>230</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar luminancia</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="quantize_amount">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>280</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="quantize_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>310</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Quantizar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="histogram_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>360</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Histograma</string>
    </property>
   </widget>
   <widget class="QPushButton" name="brightness_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>390</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Brilho</string>
    </property>
   </widget>
   <widget class="QPushButton" name="contrast_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>420</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Contraste</string>
    </property>
   </widget>
   <widget class="QPushButton" name="negative_btn">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>450</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Negativa</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1058</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
